db/schema.rb

```diff
@@ -110,9 +111,9 @@
     t.integer "failed_attempts", default: 0, null: false
     t.string "unlock_token"
     t.datetime "locked_at", precision: nil
-    t.boolean "employed"
     t.datetime "remember_created_at", precision: nil
     t.string "remember_token"
+    t.boolean "employed"
```

Comment:
this sneaked in during a rebase, but if others always find they get this change when applying a migration locally perhaps we could keep in